### PR TITLE
improve(InventoryClient): Safer dereferencing & use bnZero

### PR DESCRIPTION
**Pull request by pxrl**
The relayer now calls into the InventoryClient in new ways, and this can backfire if the relayer doesn't have an inventory config defined.

While here, sub out instances of toBN(0) in favour of the predefined bnZero. This is more memory efficient and saves creating disposable BNs.

### CODE REVIEW

**Review by nicholaspai — APPROVED** [review]

File: src/clients/InventoryClient.ts [hunk 1]
```diff
@@ -20,6 +21,8 @@ import { Deposit, FillsToRefund, InventoryConfig } from "../interfaces";
 import lodash from "lodash";
 import { CONTRACT_ADDRESSES } from "../common";
 
+const { bnZero } = sdkUtils;
```
Comment (james-a-morris) [inline src/clients/InventoryClient.ts:24]: We have a `utils/SDKUtils.ts` file for exporting from the sdk-v2 repo. Thoughts on moving this there?

**Review by james-a-morris — APPROVED** [review]
Looks great. Left one nit that I'd like to see before we merge

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/clients/InventoryClient.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/InventoryClient.ts:24]: I'll follow up with a separate PR for this, since we use it in many places.

**Pull request merged**